src/main/java/com/sonyericsson/jenkins/plugins/bfa/PluginImpl.java

```diff
@@ -133,6 +133,7 @@
 
     private int nrOfScanThreads;
     private int maxLogSize;
+    private Boolean enableBuildDescription;
```

Comment:
_nit_ there is no need for a `Boolean` object here. `boolean`s in java defaults to `false` if not set.